elastic/apm-server
# Cloud metadata

## Motivation/summary

Add `cloud` metadata to Elastic APM event schema. See https://github.com/elastic/apm/issues/256 for rationale.

## Checklist

~- [ ] I have made corresponding changes to the documentation~

## How to test these changes

1. Implement agent-side support for detecting and sending cloud metadata (https://github.com/elastic/apm/issues/256)
2. Have the agent send some transactions to the server
3. Check that the new fields are indexed

## Related issues

Closes https://github.com/elastic/apm-server/issues/3660
